cli/logs/logs.go

```diff
@@ -197,6 +203,40 @@ func printLogs(ctx context.Context, rawLogs <-chan rawLogLine, noColor bool) err
 		// Parse the logs in the windows to extract their timestamps.
 		var parsedLogs []parsedLogLine
 		for _, rawLog := range window {
+			if rawLog.readError == io.EOF {
+				// Specially handle EOF.
+
+				// If we got a message (which might be possible), try to parse
+				// it.
+				if rawLog.message != "" {
+					message, timestamp, err := parseLogLine(rawLog.message)
+					if err != nil {
+						// Don't warn here, this is reasonable.
+						message = rawLog.message
+						timestamp = rawLog.receivedAt
+					}
+
+					parsedLogs = append(parsedLogs, parsedLogLine{
+						fromContainer: rawLog.fromContainer,
+						message:       message,
+						loggedAt:      timestamp,
+					})
+				}
+
+				// If we are following, let the user know that the container
+				// terminated.
+				if handleEOF {
+					parsedLogs = append(parsedLogs, parsedLogLine{
+						loggedAt:       rawLog.receivedAt,
```

Comment:
Should we use `sort.SliceStable` so that the exit message will come after the last log?

Reply:
https://github.com/kelda/blimp/pull/3

Reply:
If there is a last log, my guess is that it will have a timestamp, so this would be a pretty rare thing. That said, it's an easy change so we may as well.